Solutions/Marain.UserNotifications.Storage.AzureStorage/Marain/UserNotifications/Storage/AzureStorage/Template/TenantedAzureBlobTemplateStoreFactory.cs

```diff
@@ -22,11 +22,13 @@ public class TenantedAzureBlobTemplateStoreFactory : ITenantedNotificationTempla
         /// The blob definition for the store. This is used to look up the corresponding configuration from.
         /// tenant.
         /// </summary>
-        public static readonly BlobStorageContainerDefinition BlobContainerDefinition = new BlobStorageContainerDefinition("templates");
+        public const string BlobContainerName = "templates";
+        private const string TemplatesV2ConfigurationKey = "StorageConfiguration__" + BlobContainerName;
```

Comment:
I think this needs to become "StorageConfiguration__Table__" like it did in the test, because that's the configuration key that any existing services running on the current version of this service will have been using. (That name was previously chosen automatically by `Corvus.Tenancy`; one of the changes is that applications now explicitly choose the configuration keys they want to use for tenanted storage. When you're migrating to the V3 libraries, applications have to specify the key names they would previously have been using implicitly.)

Reply:
Oops. My bad. I commented on the wrong file. It was `TenantedAzureTableUserNotificationStoreFactory.cs` that needed this (and that does indeed have this.)